packages/flutter/test/material/outline_button_test.dart

```diff
@@ -43,90 +43,109 @@ void main() {
-      outlineButton,
+      disabledOutlineButton,
       paints
-        // initially the interior of the button is transparent
-        ..path(color: fillColor.withAlpha(0x00))
         ..clipPath(pathMatcher: coversSameAreaAs(clipPath, areaToCompare: clipRect.inflate(10.0)))
-        ..path(color: borderColor, strokeWidth: borderWidth)
+        ..path(color: disabledBorderColor, strokeWidth: borderWidth));
+
+
+    // Pump a new widget with no onPressed to make it enabled.
+    const Key enabledKey = Key('enabled');
+    await tester.pumpWidget(
+      buildFrame(enabledKey, () {}),
     );
 
-    final Offset center = tester.getCenter(outlineButton);
+    final Finder enabledOutlineButton = find.byKey(enabledKey);
+    expect(tester.widget<OutlineButton>(enabledOutlineButton).enabled, true);
+
+    await tester.pump();
+    expect(
+      enabledOutlineButton,
+      paints
+      // initially the interior of the button is transparent
+        ..path(color: fillColor.withAlpha(0x00))
+        ..clipPath(pathMatcher: coversSameAreaAs(clipPath, areaToCompare: clipRect.inflate(10.0)))
+        ..path(color: borderColor, strokeWidth: borderWidth));
+
+    final Offset center = tester.getCenter(enabledOutlineButton);
     final TestGesture gesture = await tester.startGesture(center);
     await tester.pump(); // start gesture
     // Wait for the border's color to change to highlightedBorderColor and
     // the fillColor to become opaque.
     await tester.pump(const Duration(milliseconds: 200));
     expect(
-      outlineButton,
+      enabledOutlineButton,
       paints
         ..path(color: fillColor.withAlpha(0xFF))
         ..clipPath(pathMatcher: coversSameAreaAs(clipPath, areaToCompare: clipRect.inflate(10.0)))
-        ..path(color: highlightedBorderColor, strokeWidth: borderWidth)
-    );
+        ..path(color: highlightedBorderColor, strokeWidth: borderWidth));
 
     // Tap gesture completes, button returns to its initial configuration.
     await gesture.up();
     await tester.pumpAndSettle();
     expect(
-      outlineButton,
+      enabledOutlineButton,
       paints
         ..path(color: fillColor.withAlpha(0x00))
         ..clipPath(pathMatcher: coversSameAreaAs(clipPath, areaToCompare: clipRect.inflate(10.0)))
-        ..path(color: borderColor, strokeWidth: borderWidth)
-    );
-    debugDisableShadows = true;
+        ..path(color: borderColor, strokeWidth: borderWidth));
   });
 
-
   testWidgets('OutlineButton contributes semantics', (WidgetTester tester) async {
     final SemanticsTester semantics = new SemanticsTester(tester);
     await tester.pumpWidget(
       new Directionality(
         textDirection: TextDirection.ltr,
         child: new Material(
           child: new Center(
-            child: new OutlineButton(
-              onPressed: () { },
-              child: const Text('ABC')
-            ),
+            child:
+                new OutlineButton(onPressed: () {}, child: const Text('ABC')),
```

Comment:
Still need `new` and `const` here in Dart 2?

Reply:
Yes please explicitly specify all news and consts still.
We are working on having a lint that will help us consistently strip them out but we want the codebase to be consistent in the meantime.